tests/antithesis/compose.go

```diff
@@ -0,0 +1,190 @@
+			return nil, err
+		}
+		tlsCert, err := node.Flags.GetStringVal(config.StakingCertContentKey)
+		if err != nil {
+			return nil, err
+		}
+		signerKey, err := node.Flags.GetStringVal(config.StakingSignerKeyContentKey)
+		if err != nil {
+			return nil, err
+		}
+
+		env := types.Mapping{
+			config.NetworkNameKey:             constants.LocalName,
+			config.AdminAPIEnabledKey:         "true",
+			config.LogLevelKey:                logging.Debug.String(),
+			config.LogDisplayLevelKey:         logging.Trace.String(),
+			config.HTTPHostKey:                "0.0.0.0",
+			config.PublicIPKey:                address,
+			config.StakingTLSKeyContentKey:    tlsKey,
+			config.StakingCertContentKey:      tlsCert,
+			config.StakingSignerKeyContentKey: signerKey,
+		}
+
+		nodeName := "avalanche"
+		if i == 0 {
+			nodeName += "-bootstrap-node"
+			bootstrapIP = address + ":9651"
+			bootstrapIDs = node.NodeID.String()
+		} else {
+			nodeName = fmt.Sprintf("%s-node-%d", nodeName, i+1)
+			env[config.BootstrapIPsKey] = bootstrapIP
+			env[config.BootstrapIDsKey] = bootstrapIDs
+		}
+
+		// The env is defined with the keys and then converted to env
+		// vars because only the keys are available as constants.
+		env = keyMapToEnvVarMap(env)
+
+		services[i+1] = types.ServiceConfig{
+			Name:          nodeName,
+			ContainerName: nodeName,
+			Hostname:      nodeName,
+			Image:         nodeImageName,
+			Volumes: []types.ServiceVolumeConfig{
+				{
+					Type:   types.VolumeTypeBind,
+					Source: fmt.Sprintf("./volumes/%s/logs", nodeName),
+					Target: "/root/.avalanchego/logs",
+				},
+			},
+			Environment: env.ToMappingWithEquals(),
+			Networks: map[string]*types.ServiceNetworkConfig{
+				networkName: {
+					Ipv4Address: address,
+				},
+			},
+		}
+
+		// Collect URIs for the workload container
+		uris[i] = fmt.Sprintf("http://%s:9650", address)
+	}
+
+	workloadEnv := types.Mapping{
+		"AVAWL_URIS": strings.Join(uris, " "),
+	}
+
+	workloadName := "workload"
+	services[0] = types.ServiceConfig{
+		Name:          workloadName,
+		ContainerName: workloadName,
+		Hostname:      workloadName,
+		Image:         workloadImageName,
+		Environment:   workloadEnv.ToMappingWithEquals(),
+		Networks: map[string]*types.ServiceNetworkConfig{
+			networkName: {
+				Ipv4Address: baseNetworkAddress + ".129",
+			},
+		},
+	}
+
+	return &types.Project{
+		Networks: types.Networks{
+			networkName: types.NetworkConfig{
+				Driver: "bridge",
+				Ipam: types.IPAMConfig{
+					Config: []*types.IPAMPool{
+						{
+							Subnet: baseNetworkAddress + ".0/24",
+						},
+					},
+				},
+			},
+		},
+		Services: services,
+	}, nil
+}
+
+// Convert a mapping of avalanche config keys to a mapping of env vars
+func keyMapToEnvVarMap(keyMap types.Mapping) types.Mapping {
+	envVarMap := make(types.Mapping, len(keyMap))
+	for key, val := range keyMap {
+		// e.g. network-id -> AVAGO_NETWORK_ID
+		envVar := "AVAGO_" + strings.ToUpper(strings.ReplaceAll(key, "-", "_"))
```

Comment:
We should make `"AVAGO"` a constant exposed from the `config` package and use it here. Currently it's just used directly as `v.SetEnvPrefix("avago")`

Reply:
We could also expose the `*strings.Replacer`